spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -82,12 +82,19 @@ public class InjectedConfigUsageSample {
     private String someUrl;
 
     //The following code injects an Optional value of myprj.some.port property.
-    //Contrary to natively injecting the configured value this will not lead to a
-    //DeploymentException if the configured value is missing.
+    //Contrary to natively injecting the configured value will not lead to a
```

Comment:
I think the original grammar was closer: 
```suggestion
    //Contrary to natively injecting the configured value, this will not lead to a
```

The wording could be better but I think this was the intent of the original sentence.

Reply:
I guess it was missing the comma, so it was looking weird to me.